Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -7091,18 +7066,47 @@ static RefPtr<CSSRayValue> consumeRayShape(CSSParserTokenRange& range, const CSS
     );
 }
 
+static RefPtr<CSSValue> consumeBasicShapeRayOrBox(CSSParserTokenRange& range, const CSSParserContext& context, ConsumeRay consumeRay)
+{
+    CSSValueListBuilder list;
+    bool funcFound = false;
+    bool boxFound = false;
+    while (!range.atEnd() && !(funcFound && boxFound)) {
+        RefPtr<CSSValue> componentValue = nullptr;
```

Comment:
```suggestion
        RefPtr<CSSValue> componentValue;
```

This should silence the style checker.

Reply:
Fixed.